editor/js/Sidebar.Project.App.js

```diff
@@ -115,10 +115,6 @@ function SidebarProjectApp( editor ) {
 
 			content = content.replace( '<!-- title -->', title );
 
-			const includes = [];
-
-			content = content.replace( '<!-- includes -->', includes.join( '\n\t\t' ) );
```

Comment:
This was part of the initial VR support in published apps, see f8e0e542d4fd05fffb534bb26b086ae5eb7f136c.

If we ever want to include addons in the published app, we should use a different approach in any event. I have outlined one possible approach in https://github.com/mrdoob/three.js/pull/28187#issuecomment-2107022036.

Reply:
implemented #28636